packages/core/package.json

```diff
@@ -17,15 +17,15 @@
     "test": "echo 'test'"
   },
   "dependencies": {
-    "app-root-path": "^3.0.0",
-    "chalk": "^4.1.2",
+    "app-root-path": "^3.1.0",
+    "chalk": "^5.0.1",
     "node-datetime": "^2.1.2",
-    "node-fetch": "^2.6.0",
+    "node-fetch": "^3.2.10",
```

Comment:
Things still run fine w/ this change? 2 -> 3 was a change to make node-fetch ESM-only and iirc the lib is written in CommonJS